### ISSUE TITLE
Missing License information

### ISSUE THREAD

**Issue by pbi-qfs** [hzulla/WeakValueHashMap#1]
Please provide a license hint for your WeakValueHashMap (e.g. MIT), so that it can be safely reused.

**Comment by JosePeRa** [comment]
Any solution to the License issue???

**Comment by jimka2001** [comment]
Yes, I'm having the same issue.  I'd like to use this code in a Scala project.  The Scala project has an MIT license.   To include the WeakValueHashMap code, I had to copy the .java file into my project.  I included a comment at the top (modified the file) to include a link back to the github page where I coped the code from.  I don't know what else I can do.

**Comment by hzulla** [comment]
I'm going to add an MIT license to address this. I consider this code free for use as you wish.
